<?xml version="1.0"?>
<launch>
    <!-- Arguments -->
    <arg name="robot_param" default="$(find param_pkg)/param/param_runtime.yaml"/>

    <!-- Load param default -->
    <node pkg="param_pkg" name="dynparam_server" type="dynparam_server" respawn="false" output="screen"></node>

    <!-- control param -->
    <node pkg="param_pkg" name="control_dynparam" type="control_dynparam" respawn="false" output="screen"></node>

</launch>